maquetado/setup/scss/navbar.scss

```diff
@@ -0,0 +1,25 @@
+$navbar-height: 90px;
+$navbar-logo-height: 30px;
+
+.navbar {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 5px 4px 10px $grey;
+  display: flex;
+  height: $navbar-height;
+  justify-content: space-between;
+  position: fixed;
+  top: 0px;
```

Comment:
Always when `fixed` or `absolute` add at least 2 parameters, top and left or top and right`